src/react/components/pages/homepage/homePage.tsx

```diff
@@ -107,9 +119,19 @@ export default class HomePage extends React.Component<IHomepageProps> {
                 }
                 <Confirm title="Delete Project"
                     ref={this.deleteConfirm}
-                    message={(project: IProject) => `${strings.homePage.deleteProject.confirmation} '${project.name}'?`}
+                    message={(project: IProject) => `${strings.homePage.deleteProject.confirmation} ${project.name}?`}
                     confirmButtonColor="danger"
                     onConfirm={this.deleteProject} />
+                <Confirm title="Import Project"
+                    ref={this.importConfirm}
+                    message={(project: any) => `${strings.homePage.importProject.confirmation} ${project.file.name}
+                        ${strings.homePage.importProject.recommendation}`}
+                    confirmButtonColor="danger"
+                    onConfirm={this.convertProject} />
+                <Confirm title="Confirm Settings"
+                    ref={this.settingsConfirm}
+                    message={"Please confirm your new v2 project settings."}
+                    onConfirm={this.deleteProject} />
```

Comment:
Delete project on confirming setting?  By design?

Reply:
No, this was a mistake. I've changed this to a message box with no onConfirm method